buf/registry/module/v1/label_service.proto

```diff
@@ -167,6 +167,9 @@ message ListLabelHistoryRequest {
     (buf.validate.field).string.uuid = true,
     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   ];
+  // Whether or not to only return Commits that represent a change in the Protobuf schema
```

Comment:
I think you mean this:
```suggestion
  // Whether or not to only return Commits that represent a change in Content
```

If we _do_ mean schema, what is included in that? Is a change to a comment also considered a schema change? Change to the license file?